src/core/json/lexer_impl.cc

```diff
@@ -6,8 +6,6 @@
 
 #include <absl/strings/str_cat.h>
 
-using namespace std;
```

Comment:
DO NOT USE `using namespace std` in source files. 

It did not pull `std::string` on the function below.... :facepalm:

Reply:
Basically stop triggering `ADL` unless you are writing customization points.

Reply:
I was `wrong`, it's not `ADL`. It's a false positive and I stepped through it and it's an `std::string`. 

Still I don't like pulling namespaces in source files :stuck_out_tongue: